Packages/com.unity.inputsystem/InputSystem/Plugins/DualShock/DualShockGamepadHID.cs

```diff
@@ -147,7 +147,7 @@ public static DualSenseHIDBluetoothOutputReport Create(DualSenseHIDOutputReportP
     /// Structure of HID input reports for PS4 DualShock 4 controllers.
     /// </summary>
     [StructLayout(LayoutKind.Explicit, Size = 32)]
-    internal struct DualShock4HIDInputReport : IInputStateTypeInfo
+    public struct DualShock4HIDInputReport : IInputStateTypeInfo
```

Comment:
@Rene-Damm should I keep it internal as per https://unity.slack.com/archives/GAX892FTJ/p1642771810095500 ?

Reply:
Fixed